### PR TITLE
style for home page

### PR DESCRIPTION
**Pull request by Pagehey**
Add style for h1 (beautiful blue)
Add container around content

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-<p>Find me in app/views/pages/home.html.erb</p>
-
-<% if user_signed_in? %>
-  <%= current_user %>
-  <%= current_user.email %>
+<div class="container">
+  <h1 class="text-primary">Welcome here !</h1>
+  <% if user_signed_in? %>
+    <%= current_user %>
+    <%= current_user.email %>
```
Comment (Pagehey) [inline app/views/pages/home.html.erb:5]: do we need this ?

Comment (Lomig) [inline app/views/pages/home.html.erb:5]: Do we need to show the email address instead of the name?

Comment (Pagehey) [reply app/views/pages/home.html.erb:5]: We don't have the name yet on users table

Comment (Lomig) [reply app/views/pages/home.html.erb:5]: Let's add that as a user Story!

**Pull request merged**